cluster/addons/fluentd-elasticsearch/fluentd-es-ds.yaml

```diff
@@ -24,14 +24,19 @@ spec:
       serviceAccountName: fluentd-es
       containers:
       - name: fluentd-es
-        image: gcr.io/google_containers/fluentd-elasticsearch:1.23
+        image: gcr.io/google_containers/fluentd-k8s-es:1.23
         command:
           - '/bin/sh'
           - '-c'
           - '/usr/sbin/td-agent $FLUENTD_ARGS'
         env:
         - name: FLUENTD_AGRS
           value: -q
+        # TODO: Make into secret!!
```

Comment:
@coffeepac That is correct, since authentication is enabled by default for ES 5.5.